mne/time_frequency/tfr.py

```diff
@@ -344,36 +616,17 @@ def cwt(X, Ws, use_fft=True, mode='same', decim=1):
     return tfrs
 
 
-def _time_frequency(X, Ws, use_fft, decim):
-    """Aux of time_frequency for parallel computing over channels
-    """
-    decim = _check_decim(decim)
-    n_epochs, n_times = X[:, decim].shape
-    n_frequencies = len(Ws)
-    psd = np.zeros((n_frequencies, n_times))  # PSD
-    plf = np.zeros((n_frequencies, n_times), np.complex)  # phase lock
-
-    mode = 'same'
-    tfrs = _cwt(X, Ws, mode, decim=decim, use_fft=use_fft)
-
-    for tfr in tfrs:
-        tfr_abs = np.abs(tfr)
-        psd += tfr_abs ** 2
-        plf += tfr / tfr_abs
-    psd /= n_epochs
-    plf = np.abs(plf) / n_epochs
-    return psd, plf
-
-
 @verbose
+@deprecated("This function will be removed in mne 0.14; use mne.time_frequency"
+            ".timefreq_transform() instead.")
```

Comment:
Put deprecated before verbose and it should fix the docstring check, we introspect for deprecated decorator but not dep under verbose